core/templates/pages/exploration-editor-page/changes-in-human-readable-form/changes-in-human-readable-form.component.html

```diff
@@ -49,70 +57,18 @@
             </span>
           </div>
 
-          <div *ngSwitchCase="'answer_groups'">
-            <div *ngIf="lostChange.getRelativeChangeToGroups() === 'added'" class="state-edit-desc answer-group">
-              <strong>Added answer group: </strong>
-              <p class="sub-edit">
-                <i>Destination: </i>{{ lostChange.newValue.outcome.dest }}
-              </p>
-
-              <div class="sub-edit">
-                <i>Feedback: </i>
-                <div class="feedback">
-                  {{ lostChange.newValue.outcome.feedback.html }}
-                </div>
-              </div>
-
-              <div class="sub-edit" *ngIf="lostChange.newValue.rules.length > 0">
-                <i>Rules: </i>
-                <ol class="rules-list">
-                  <li *ngFor="let rule of lostChange.newValue.rules">
-                    <p>Type: {{ rule.type }}</p>
-                    <p>Value:
-                      <span> </span>
-                      <ng-container *ngFor="let input of rule.inputs | keyvalue; let last = last">
-                        {{ last ? input.value: (input.value + ',') }}
-                      </ng-container>
-                    </p>
-                  </li>
-                </ol>
-              </div>
```

Comment:
Seems like we are loosing some information here. We seem to be printing more details regarding "added answer groups" in the old code but the new one just says "Added answer group"? Please explain this change.

Reply:
These details which we were showing here were according to the very old version of the answer groups. Since then the answer groups have changed significantly and this template was returning errors. Now the answer groups occur in the list so at present we are not handling these cases. That's why fixed the errors by changing the way the changes in answer group can be shown.

Reply:
Ah, I see. Thanks for clarifying!